packages/vitepress-theme/src/mixins.scss

```
@@ -43,11 +43,34 @@
   );
   --vp-home-hero-image-filter: blur(44px);
 
-  @media (min-width: 960px) {
+  @include breakpoint(l, min) {
     --vp-home-hero-image-filter: blur(68px);
   }
 
-  @media (min-width: 640px) {
+  @include breakpoint(s, min) {
     --vp-home-hero-image-filter: blur(56px);
   }
 }
+
+/**
+ * Applies CSS only to the given VitePress breakpoint.
+ * @param {string} $name - name of the VitePress breakpoint. One of: xs, s, m, l, xl or xxl
+ * @param {string} $min-max - Whether the breakpoint should be considered as min or max width.
+ *
+ * @example breakpoint(m, min)
+ * @example breakpoint(m, max)
+ */
+@mixin breakpoint($name, $min-max) {
```

imho, in terms of readability, it would go smoother with min/max as first param
```suggestion
@mixin breakpoint($min-max, $name) {
```
so then you read "min m" -> minimally m